docs/sources/explore/explore-inspector.md

```diff
@@ -0,0 +1,82 @@
++++
+title = "Inspector in Explore"
+weight = 400
++++
+
+# Inspector in Explore
+
+The inspector helps you understand and troubleshoot your queries. You can inspect the raw data, export that data to a comma-separated values (CSV) file, export log results in TXT format, and view query requests.
+
+## Inspector UI
+
+The inspector consists of four tabs:
+
+- **Data tab -** Shows the raw data returned by the query.
+- **Stats tab -** Shows how long your query takes and how much it returns.
+- **JSON tab -** Allows you to view and copy the data JSON and data frame structure JSON.
+- **Query tab -** Shows you the requests to the server sent when Grafana queries the data source.
+
+## Inspector tasks
+
+Tasks you can perform in the inspector are described below.
+
+### Open the Inspector
+
+1. In Grafana, navigate to the Explore and run the query you would like to inspect.
+1. Click the **Inspector** button.
+
+The inspector pane opens on the bottom of the screen.
+
+### Inspect raw query results
+
+View raw query results in a table. This is the data returned by the query.
+
+1. Open the inspector and then click the **Data** tab.
+1. If you have run multiple queries or queries multiple nodes, then you have additional options.
+
+- **Show data frame -** Choose which result set data you want to view.
+  - **Series joined by time -** View raw data from all your queries at once, one result set per column. Click a column heading to reorder the data.
```

Comment:
```suggestion
- **Series joined by time:** View the raw data from all of your queries at once, one result set per column. You can click a column heading to sort the data.
```

Reply:
I did not see this option in https://play.grafana.org/explore?orgId=1&left=%5B%22now-1h%22,%22now%22,%22demo.robustperception.io%22,%7B%22exemplar%22:true,%22expr%22:%22ALERTS%22,%22requestId%22:%22Q-61eb23f3-97e4-4ea2-8247-3eb3bb2ec094-0A%22%7D%5D

Reply:
It is there: ![image](https://user-images.githubusercontent.com/30407135/116858247-5faabd00-abfe-11eb-8228-c32480330494.png)